index.js

```diff
@@ -33,6 +33,8 @@
   var registeredComponents = [];
   var handlers = [];
 
+  var IGNORE_CLASS_REGEXP = new RegExp('ignore-react-onclickoutside');
```

Comment:
no need for this to be a regexp, actually, let's just make this `var IGNORE_CLASS = "ignore-react-onclickoutside";`, see below